starboard/android/shared/audio_track_audio_sink_type.cc

```diff
@@ -188,7 +188,7 @@ void AudioTrackAudioSink::SetPlaybackRate(double playback_rate) {
 
 // static
 void* AudioTrackAudioSink::ThreadEntryPoint(void* context) {
-  pthread_setname_np(pthread_self(), "audio_track_audio_out");
+  pthread_setname_np(pthread_self(), "audio_track_aud");
```

Comment:
Maybe we should add a comment here that "truncate name as "audio_track_aud" due to `pthread_setname_np` is limited to 16 characters". Feel free to rephrase it.

Reply:
Updated to `audio_track_out`.